tox.ini

```diff
@@ -102,6 +102,31 @@ commands =
 
     {envbindir}/coverage run "{envbindir}/trial"  {env:TRIAL_FLAGS:} {posargs:tests} {env:TOXSUFFIX:}
 
+[testenv:synapse_port_db]
```

Comment:
I'm a bit unconvinced that `tox.ini` is the right place for this. Ultimately what you've written is pretty specific to the buildkite configuration - or at least, the docker images that buildkite will use to run the build - so it's not like anyone can just run `tox -e synapse_port_db` from outside the right docker config.

I think a shell script (possibly in .buildkite) might be a simpler solution.

Reply:
I don't really see how it's that specific to buildkite, and I believe people can run that `tox` call from their own machine from outside any docker config (because I'm doing it in order to test that it does the right thing).